Makefile

```diff
@@ -1,7 +1,7 @@
-NS = spotify
+NS = default
 
 SERVICE_NAME = bigtable-autoscaler
-SERVICE_IMAGE_VERSION = latest
+SERVICE_IMAGE_VERSION ?= 0.0.1 
```

Comment:
What does `?=` mean here?

Reply:
use `0.0.1` if the value isn't already defined

Reply:
TIL. Thanks!